### PR TITLE
Fix: Grid component header and cells

### PR DESCRIPTION
**Pull request by patrick-ivers**
#### Changes
- Make .grid-component-cell-* class names consistent (incl. change ...-header-right to ...-header-toggle)
- Center content in grid addon cells
- Rename fontSize.scss and fontWeight.scss to font-size.scss and font-weight.scss for consistency

### Screenshots
screen shot 2016-06-22 at 4 42 26 pm

### CODE REVIEW

File: src/components/adslotUi/ListPickerPureComponent.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const ListPickerPureComponent = ({
             <GridCell stretch>
               {itemHeaders.left}
             </GridCell>
-            <GridCell classSuffixes={['header-right']}>
+            <GridCell classSuffixes={['header-toggle']}>
               {itemHeaders.right}
```
Comment (NamrataTrivedi) [inline src/components/adslotUi/ListPickerPureComponent.js]: should we change this to itemHeaders.right to toggle? I think we are no longer using 'right', its either toggle or addon.

Comment (patrick-ivers) [inline src/components/adslotUi/ListPickerPureComponent.js]: Good question. I'll find out

**Comment by vinteo** [comment]
LGTM

**Comment by NamrataTrivedi** [comment]
LGTM :wrench:

**Comment by omgaz** [comment]
LGTMerging 🍜

**Pull request merged**